consumer-test/s4sdk-neo/runTests.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+
+LIBRARY_VERSION_UNDER_TEST=$(git log --format="%H" -n 1)
+REPOSITORY_UNDER_TEST=${TRAVIS_REPO_SLUG:-SAP/jenkins-library}
+
+rm -rf workspace
+git clone -b consumer-test-neo https://github.com/sap/cloud-s4-sdk-book workspace
+cp -f jenkins.yml workspace
+cd workspace
+sed -i -e "s:__REPO_SLUG__:${REPOSITORY_UNDER_TEST}:g" jenkins.yml
```

Comment:
Havving a comment here, would also be nice.
Furthermore, we now have duplicated code between neo and cloud foundry.